src/Components/EmailNotificationForm/EmailNotificationForm.js

```diff
@@ -0,0 +1,271 @@
+import { Component } from "react";
+import './EmailNotificationForm.css';
+
+class EmailNotificationForm extends Component {
+  constructor() {
+    super();
+    this.state = {
+      first_name: '',
+      last_name: '',
+      state: '',
+      email: '',
+      language: '',
```

Comment:
Do we want to have a default language?

Reply:
For this radio button, I think the way that the functionality is set up right now will work.